components/asset.rst

```diff
@@ -0,0 +1,251 @@
+.. index::
+   single: Asset
+   single: Components; Asset
+
+The Asset Component
+===================
+
+   The Asset component manages URL generation and versioning of web assets such
+   as CSS stylsheets, JavaScript files and image files.
+
+Installation
+------------
+
+You can install the component in two different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/asset`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/Asset).
+
+Usage
+-----
+
+Asset Packages
```

Comment:
In my opinion, the real problem was the lack of an introduction for the asset. In ac58089 I briefly explain the benefits that the Asset component provides comparing it with no using any asset management.